pkg/combustion/templates/k8s-vip.yaml.tpl

```diff
@@ -27,7 +27,7 @@ metadata:
   name: endpoint-copier-operator
   namespace: endpoint-copier-operator
 spec:
-  repo: https://suse-edge.github.io/endpoint-copier-operator
+  repo: https://suse-edge.github.io/charts
   chart: endpoint-copier-operator
```

Comment:
This may be my lack of Helm knowledge, but is there a way to specify the version here? Feels fragile to not have it pinned for a production release.